tests/perf/client-performance-test.md

```diff
@@ -0,0 +1,50 @@
+# Setting up and Running Performance Tests
```

Comment:
Probably just name this file `README.md` so it will display automatically when going to `perf` directory

Reply:
Good suggestion. Updated.